libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -405,4 +445,25 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       month: this.month,
     };
   }
+
+  // TODO: Should this be based on new Date() (so deterministically today) or this.todayDate?
+  // TODO: Is an if else statement more readable here?
+  private getDateFromNavigableYear(navigableYear: number | Date): Date {
+    return navigableYear instanceof Date
+      ? navigableYear
+      : new Date(
+          (this.todayDate ? this.todayDate : new Date()).getFullYear() + navigableYear,
+          0,
+          1
+        );
+  }
+
+  // TODO: Perhaps moment can already do this ? ? ?
+  private findYearsBetweenDates(startDate: Date, endDate: Date): string[] {
+    const yearsBetweenDates = [];
+    for (let year = startDate.getFullYear(); year <= endDate.getFullYear(); year++) {
+      yearsBetweenDates.push(year.toString());
+    }
+    return yearsBetweenDates;
+  }
```

Comment:
Can be achieved trough regular js:
```suggestion
  private findYearsBetweenDates(startDate: Date, endDate: Date): string[] {
   // Ensure years are ordered correctly if parameters are switched:  
    const [startYear, endYear] = [startDate.getFullYear(), endDate.getFullYear()].sort();
    const numberOfYears = endYear - startYear;
    return Array.from({length: numberOfYears + 1}, (_, i) => (startYear + i).toString());
  }
```